components/ws-manager/pkg/manager/testdata/cdwp_duplicate_featureflag.golden

```diff
@@ -1,236 +1,12 @@
 {
     "reason": {
         "metadata": {
-            "name": "ws-test",
-            "namespace": "default",
-            "creationTimestamp": null,
-            "labels": {
-                "app": "gitpod",
-                "component": "workspace",
-                "gitpod.io/networkpolicy": "default",
-                "gpwsman": "true",
-                "headless": "false",
-                "metaID": "foobar",
-                "owner": "tester",
-                "workspaceID": "test",
-                "workspaceType": "regular"
-            },
-            "annotations": {
-                "container.apparmor.security.beta.kubernetes.io/workspace": "unconfined",
-                "gitpod.io/requiredNodeServices": "ws-daemon,registry-facade",
-                "gitpod/admission": "admit_owner_only",
-                "gitpod/contentInitializer": "GmcKZXdvcmtzcGFjZXMvY3J5cHRpYy1pZC1nb2VzLWhlcmcvZmQ2MjgwNGItNGNhYi0xMWU5LTg0M2EtNGU2NDUzNzMwNDhlLnRhckBnaXRwb2QtZGV2LXVzZXItY2hyaXN0ZXN0aW5n",
-                "gitpod/id": "test",
-                "gitpod/imageSpec": "Cm1ldS5nY3IuaW8vZ2l0cG9kLWRldi93b3Jrc3BhY2UtYmFzZS1pbWFnZXMvZ2l0aHViLmNvbS90eXBlZm94L2dpdHBvZDo4MGE3ZDQyN2ExZmNkMzQ2ZDQyMDYwM2Q4MGEzMWQ1N2NmNzVhN2FmEjRldS5nY3IuaW8vZ2l0cG9kLWNvcmUtZGV2L2J1aWQvdGhlaWEtaWRlOnNvbWV2ZXJzaW9u",
-                "gitpod/never-ready": "true",
-                "gitpod/ownerToken": "%7J'[Of/8NDiWE+9F,I6^Jcj_1\u0026}-F8p",
-                "gitpod/servicePrefix": "foobarservice",
-                "gitpod/traceid": "",
-                "gitpod/url": "test-foobarservice-gitpod.io",
-                "gitpod/withUsernamespace": "true",
-                "prometheus.io/path": "/metrics",
-                "prometheus.io/port": "23000",
-                "prometheus.io/scrape": "true",
-                "seccomp.security.alpha.kubernetes.io/pod": "localhost/workspace-default"
-            }
+            "creationTimestamp": null
```

Comment:
This looks like a broken test to me. Looks like the "double feature flag" we were using was "user_namespace".

Reply:
fixed